recipe-server/client/control_new/utils/api.js

```diff
@@ -0,0 +1,75 @@
+import * as Cookie from 'js-cookie';
+
+
+export default class APIClient {
```

Comment:
Can you elaborate on why you made this a class? After internal debate, it makes sense to do this, but I'd be interested to hear your thoughts on it.